.github/workflows/Coverage.yml

```diff
@@ -0,0 +1,41 @@
+name: Jest Coverage
```

Comment:
Then we can call this name Generate badges.

Reply:
done